CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # create-t3-app
 
+## 4.4.1
+
+### Patch Changes
+
+- changed ui for template
```

Comment:
Please update to match what was done

Reply:
oh this is coming from the main branch I checkout from out